blog/_posts/2016-11-23-release-notes-4.5.0-RC1.md

```diff
@@ -0,0 +1,61 @@
+---
+ layout: blog
+ title: Scala IDE 4.5.0-RC1 is out!
+ author: Scala IDE team
+ disqus: false
+ ---
```

Comment:
formatting is off here. I can't render correctly. You need to remove the space at the beginning of the lines.